docs/control-structures.rst

```diff
@@ -171,6 +214,9 @@ throws an exception or goes out of gas.
     external functions happen after any changes to state variables in your contract
     so your contract is not vulnerable to a reentrancy exploit.
 
+...note::
+    A function call from one contract to another does not create its own transaction, you have to look in the overall transaction. This is also the reason why several block explorer do not show Ether sent between contracts correctly.
```

Comment:
I would drop the "you have to look" - it assumes that you are searching for something. Perhaps something like "... its own transaction, it is just a message call as part of the overall transaction."